.github/workflows/main.yml

```diff
@@ -36,12 +36,16 @@ jobs:
     - name: Run lint for /client
       run: cd client && npm run lint
 
+    - name: Run tests for /client
+      run: cd client && npm run test
+
```

Comment:
This might break the build since tests currently run in watch mode (you have to quit manually after running tests). I think there's a different mode if running in CI. 

https://stackoverflow.com/questions/58408035/whats-the-difference-between-watchall-false-and-ci-true

Reply:
Yeah it broke it, lol, let me investigate in a bit :D.

Reply:
yea I just added the fix haha thanks!